templates/ts-apollo-mongodb-datasync-backend/model/Note.graphql

```diff
@@ -3,7 +3,7 @@
 @datasync
 """
 type Note {
-  id: ID!
+  _id: GraphbackObjectID!
```

Comment:
This will mean that the [client tempalate](https://github.com/aerogear/graphback/tree/2b355e62f73aae04de10878701e4fb595fcfb482/templates/ts-react-apollo-client) will need some updates i.e running a `yarn graphback generate` once downloaded for it to work. We have that in outputted in the console, but I thought I could point out this change.

Reply:
Yes. We are missing graphql codegen types

Reply:
Yes. Right now our scalars will be `any` typed, I guess we can add a [`scalar map`] (https://graphql-code-generator.com/docs/plugins/typescript-type-graphql#scalars-scalarsmap) in https://github.com/aerogear/graphback/blob/18ca16691de88feda671f2f1271ab81ec0c5dd42/templates/ts-react-apollo-client/codegen.yml